.github/workflows/python-test.yml

```diff
@@ -221,3 +221,29 @@ jobs:
           cd python
           source alibiexplainer/.venv/bin/activate
           pytest --cov=alibiexplainer ./alibiexplainer
+
+      # ----------------------------------------Huggingface Server Unit Tests------------------------------------------------
+      # @TODO this needs github action large runner
```

Comment:
This is new and might help here https://github.blog/2024-01-17-github-hosted-runners-double-the-power-for-open-source/

Reply:
awesome! going to try out!

Reply:
nice looks like now I can run the hugging face tests

Reply:
Great!